x/wasm/internal/keeper/keeper.go

```diff
@@ -93,7 +129,11 @@ func (k Keeper) Create(ctx sdk.Context, creator sdk.AccAddress, wasmCode []byte,
 	}
 	store := ctx.KVStore(k.storeKey)
 	codeID = k.autoIncrementID(ctx, types.KeyLastCodeID)
-	codeInfo := types.NewCodeInfo(codeHash, creator, source, builder)
+	if instantiateAccess == nil {
+		defaultAccessConfig := k.getInstantiateAccessConfig(ctx).With(creator)
+		instantiateAccess = &defaultAccessConfig
+	}
+	codeInfo := types.NewCodeInfo(codeHash, creator, source, builder, *instantiateAccess)
```

Comment:
This InstantiateInfo doesn't seem to be read anywhere.

I guess that change (permission on instantiate) is for another PR?

Reply:
yes, will push the other PR in a minute